tests/src/integration/objects/cluster.hpp

```diff
@@ -216,11 +216,18 @@ class Cluster : public Object<CassCluster, cass_cluster_free> {
    *               otherwise (default: true)
    * @return Cluster object
    */
+  #if defined(__GNUC__) || defined(__INTEL_COMPILER)
+  #  pragma GCC diagnostic push
+  #  pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+  #endif
   Cluster& with_hostname_resolution(bool enable = true) {
     EXPECT_EQ(CASS_OK, cass_cluster_set_use_hostname_resolution(
                            get(), (enable == true ? cass_true : cass_false)));
     return *this;
   }
+  #if defined(__GNUC__) || defined(__INTEL_COMPILER)
+  #  pragma GCC diagnostic pop
+  #endif
```

Comment:
We _might_ be able to do away with this function all together.  It's only used in some of the DSE tests and those tests appear to pass without them, at least against DSE 6.7.7.  The code above seems less intrusive than potentially changing how a test behaves so I decided to start with this... but I could be convinced otherwise.